### ISSUE TITLE
A variant with a better accented focus ring styling?

### ISSUE THREAD

**Issue by moniuch** [Curve/Nord-Dark-Pro#2]
I am a huge fan of Nord Dark Pro (using it in JetBrains as well as in VSCode). But if there's anything I am missing from it is a better, more prominent styling of focus rings.

I have compared a couple of dark themes in VSCode to see how they support focus rings, and they draw a colored line - both around the pane/area and the input controls.

**Nord Dark Pro**

![image](https://user-images.githubusercontent.com/681956/227765342-c50f835b-9e61-4297-a60c-11a77bdf546b.png)

Input focus:

![image](https://user-images.githubusercontent.com/681956/227765736-149f9d1d-d60f-4b21-a6ad-343a5fbb52f8.png)

**VS Code Dark**

![image](https://user-images.githubusercontent.com/681956/227764767-ea9d4efe-4e1a-4f01-b414-3f50f312b3ee.png)

![image](https://user-images.githubusercontent.com/681956/227765662-b72f71f3-e994-4122-bc89-0c39aa5de210.png)

Could you make a variant of Nord Pro with a bit more prominent focus rings?

When it comes to focus rings they are way too subtle to be used as a focus indicator. They can be confused with other lines (eg. code folding or the terminal pane divider). I'm sure you could choose a nice non-grey color that would still blend in nicely in the aesthetics but also provide a better feedback.

Just to clarify - I'm sure you understand this is not about the aesthetics only. It's about productivity and a11y. I've been trying to use VSCode without a mouse, and the Git panel caused me a lot of problems because of various focus areas involved in the flow.

**Comment by Curve** [comment]
I'll have a look later ^^

Just out of curiosity, how did you get this theme running in jetbrains?

**Comment by moniuch** [comment]
OK, sorry, I goofed and confused the names, what I am using in JB is actually called Nord, it has a very similar color palette :)
